helper_methods.rb

```diff
@@ -338,19 +337,55 @@ def add_revision_statement(uri, text)
 end
 
 #add a maintenance statement to agent records
-def add_maintenance_history(uri, text)
-  record = @client.get(uri).parsed
+def add_maintenance_history(record, text)
+  #record = @client.get(uri).parsed
```

Comment:
Is this being kept so it can be reverted in the future? Maybe a comment would be helpful?